packages/itwinui-css/src/side-navigation/side-navigation.scss

```diff
@@ -114,7 +114,7 @@ $iui-side-navigation-icon-margins: calc(1.5 * var(--iui-size-m));
       border-color: var(--iui-color-background-border);
     }
 
-    > .iui-button-icon:not(.iui-avatar) svg {
+    .iui-button-icon:not(.iui-avatar) svg {
```

Comment:
Removed the `>`, since now `span.iui-notification-marker` comes between `.iui-sidenav-button` and `.iui-button-icon`, as on [L132-145](https://github.com/iTwin/iTwinUI/pull/768/files#diff-324482a88e5ac05671e3d09fda294087190f1cef2c7af7631b34b7df82e9da86R132-R145).